style.css

```diff
@@ -44,7 +45,7 @@ border-radius: 50%;
 .menu__item {
   position: relative;
   margin-right: 26px;
-  font: 500 18px/21px Roboto, arial, helvetica, sans-serif;
+  font: 500 18/21px Roboto,arial,helvetica,sans-serif;
```

Comment:
Codacy found an issue: [Expected single space after "," in a single-line list (value-list-comma-space-after)](https://app.codacy.com/gh/WebHeroSchool/HTML_CSS_2.0-Artem1998/pullRequest?prid=6242157)